ios/brave-ios/Sources/BrowserMenu/Identifiers.swift

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveWallet
+import Foundation
+
+extension Action.Identifier {
+
+  public static let vpn: Self = .init(
+    id: "ToggleVPN",
+    title: Strings.ActionTitles.vpn,
+    braveSystemImage: "leo.product.vpn",
+    defaultRank: 100,
```

Comment:
What are these numbers and where do they come from? 
They don't seem to be sequential from `1...10` for example. They jump from 100 to 1600 to 1700, etc.
We should probably just add a comment explaining the rank so if someone adds a new item, they understand it :)

Reply:
Good call, they're coming from a Figma list at the moment. They're sequential in 100s at the moment, specifically with enough gap that lets us add more items in between in the future. It was basically either I order the actual Identifiers by rank or by type and I chose type (feature links > page actions > activity share sheet actions). I'll add more information on this at the top of this file